### ISSUE TITLE
Could I run pretraining on multi gpu?

### ISSUE THREAD

**Issue by hairzooc** [brightmart/albert_zh#50]
Hi,
I've been taking a look at the codes implemented and couldn't find the way to train on multi gpu.
Is there some way to do that? or you have any plans to implement this feature.
Please let me know and thanks in advance :)

**Comment by 652994331** [comment]
yes, you can, to do this , you need to modify the run_classifier.py and optimization.py.

**Comment by 652994331** [comment]
we use mirrored distribute strategy to use multi gpu to do the pretrtain using our own data. 2 gpus could make the speed x2. you also can deploy horovod, a new distribute strategy.
